holdings_maintenance/pds4/pds4linkshelf.py

```
@@ -0,0 +1,1268 @@
+                if basename.startswith('.'):    # skip invisible files
+                    logger.invisible('Invisible file skipped', abspath)
+                    continue
+
+                # collection_basename_dict: a dictonary with the abspath of a collection
+                # csv file as the key and the set of basenames of its corresponding
+                # entries as the value.
+                # Create collection_basename_dict and use it to check whether a file
+                # is listed in the csv later.
+                if (basename.startswith('collection') and
+                    basename.endswith('.csv') and
+                    not abspath in collection_basename_dict):
```

Flake8 (7.3.0)

[error] test for membership should be 'not in'

(E713)